docs/configuration.md

```diff
@@ -155,12 +182,13 @@ It's a slice of `Key` and `Value`.
 !!! example
     ```yaml
     target:
-      type: aws
-      aws:
-        ecrOptions:
-          tags:
-            - key: cluster
-              value: myCluster
+      registry:
```

Comment:
@Caomoji Does `target` have other attributes besides `registry`? If not, I would like to avoid having another level underneath because:
* simplicity, avoiding nesting if not strictly necessary
* backwards compatibility

Let me know your thoughts.

Would it help to change the type of `Target` to `Registry`?

https://github.com/estahn/k8s-image-swapper/blob/78f64244d77aaab14f78ef82174094616f4c9c77/pkg/config/config.go#L45

Example:
```
Target Registry `yaml:"target"`
```

Reply:
As of now I agree it makes more sense to directly set the registry fields under `target`, I did the update :+1: